microservices/product-search/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM php:7.2-fpm-alpine
```

Comment:
Is there any way to not have this file in root of the project? It is different from every dockerfile location in project

Reply:
I know, but the files available by Dockerfile are dependent on its location. For example, if you build Dockerfile located in microservices/product-search/conf/Dockerfile using docker build command or docker-compose build command, you cannot access any files located in elsewhere than in that directory.